integration_tests/e2e/induction/updateCheckYourAnswersChangeLinks.cy.ts

```diff
@@ -44,14 +43,13 @@ context(`Change links on the Check Your Answers page when updating an Induction`
       .selectHopingWorkOnRelease(HopingToGetWorkValue.NOT_SURE)
       .submitPage()
 
-    // Change Reasons For Not Wanting To Work
+    // Change affecting ability to work
     Page.verifyOnPage(CheckYourAnswersPage)
-    checkYourAnswersPage
-      .clickReasonsForNotWantingToWorkChangeLink()
+      .clickFactorsAffectingAbilityToWorkChangeLink()
       .hasBackLinkTo(`/prisoners/${prisonNumber}/induction/check-your-answers`)
-      .deSelectReasonNotToGetWork(ReasonNotToGetWorkValue.HEALTH)
-      .chooseReasonNotToGetWork(ReasonNotToGetWorkValue.RETIRED)
-      .chooseReasonNotToGetWork(ReasonNotToGetWorkValue.NO_REASON)
+      .deSelectAffectAbilityToWork(AbilityToWorkValue.LIMITED_BY_OFFENCE)
+      .chooseAffectAbilityToWork(AbilityToWorkValue.RETIRED)
+      .chooseAffectAbilityToWork(AbilityToWorkValue.REFUSED_SUPPORT_WITH_NO_REASON)
```

Comment:
maybe rename to select/deSelect rather than choose/deSelect?

Reply:
You make a valid point, but these methods are not new in this PR and are in use by quite a few cypress tests
But I do agree the naming style is confusing/inconsistent 👍 

Given this PR is already absolutely massive I'm thinking this PR is probably not the best place to do this refactoring, but we can definitely create a ticket to address it as tech-debt.
Does that work for you?

Reply:
Not fussed TBH. Figured it'd just be it a shift+F6 and done job but if not no-worries.

Reply:
@james-baker-digital-justice ; I've done the rename and pushed it in commit 4dcdb6715d8746f267dce475d763182c460d74b9
Though it has added 20 arguably unrelated changed files to this (already very big) PR. It has taken it from 63 changes files up to 83 !

Up to you whether you are happy to take this change in this PR; if not we can easily revert that commit 👍